src/clients/FlexbaseClient.Company.ts

```diff
@@ -49,4 +67,31 @@ export class FlexbaseClientCompany extends FlexbaseClientBase {
             return { success: false, error: 'Unable to get company payments', payments: [] };
         }
     }
+
+    async getCompanyData(): Promise<CompanyResponse> {
+        try {
+            const response = await this.client.url(`/tenant`).get().json<ResponseCompanyData>();
+
+            return {
```

Comment:
We should check here the success of the response so based on that we could return something 

```
if (!response.success) {
       this.logger.error('While trying to get a banking deposit account, an unhandled exception was thrown', 
        response.error);
     }

    return { //... };
```

Reply:
This endpoint doesn't return a `success`